.travis.yml

```diff
@@ -1,27 +1,40 @@
 language: ruby
-sudo: false
+os: linux
 dist: bionic
 
-services:
-  - postgresql
-
 cache:
   bundler: true
 
 addons:
-  postgresql: "9.6"
+  postgresql: "11"
+  apt:
+    update: true
+    packages:
+      - postgresql-11
+      - postgresql-client-11
+      - postgresql-server-dev-11
 
 before_install:
-  - gem update --system 3.1.2
-  - gem install bundler
+  - sudo cp /etc/postgresql/{9.6,11}/main/pg_hba.conf
+  - sudo service postgresql restart 11
+  - gem update --system 3.1.4
+  - gem install bundler:2.1.4
+
+before_script:
+  - bundle exec rails app:db:create app:db:migrate app:db:schema:load
 
 rvm:
   - 2.5.8
   - 2.6.6
 
 env:
   global:
+    - PGHOST=127.0.0.1
+    - PGUSER=postgres
+    - PGPORT=5432
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+    - RAILS_ENV=test
```

Comment:
Move back into Rakefile `ci` task?

Reply:
Appears this gets ignored when I have run the rake task in travis. So I explicitly set it as an env variable.

Reply:
Also just tested it locally. As long as you don't have an instance of the test solr_wrapper running when you start it, it works